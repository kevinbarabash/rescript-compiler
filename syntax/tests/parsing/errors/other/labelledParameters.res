let f = (x, y=2, z) => x + y + z

type f = (x: int, y: int) => int
