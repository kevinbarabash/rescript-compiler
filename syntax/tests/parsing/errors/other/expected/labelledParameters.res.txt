
  Syntax error!
  tests/parsing/errors/other/labelledParameters.res:1:13-14

  1 │ let f = (x, y=2, z) => x + y + z
  2 │ 
  3 │ type f = (x: int, y: int) => int

  A labeled parameter starts with a `~`. Did you mean: `~y`?


  Syntax error!
  tests/parsing/errors/other/labelledParameters.res:3:11

  1 │ let f = (x, y=2, z) => x + y + z
  2 │ 
  3 │ type f = (x: int, y: int) => int
  4 │ 

  A labeled parameter starts with a `~`. Did you mean: `~x`?

let f x ?(y= 2)  z = (x + y) + z
type nonrec f = x:int -> y:int -> int